wiedza_i_zycie/ui/src/app/app-routing.module.ts

```diff
@@ -2,7 +2,8 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
 import { EditionsComponent } from './editions';
-import { MainViewComponent } from './main-view/main-view.component'
+// import { ArticlesComponent } from './articles';
```

Comment:
zakomentowane rzeczy od razu proszę usuwaj. Wiem, że pewnie myślisz, ale za jakiś czas to będę musiał dodać, jednak musi pamiętać, że przy każdej iteracji staramy się utrzymać kod w stanie jak najlepszym. Jedynym wyjątkiem od reguły usuwania zakomentowanego kodu to sytuacja w której np. postanawiasz eksperymentować z jakimś i na jakiś czas komentujesz jedną z alternatyw